packages/jaeger-ui/src/utils/config/get-config.tsx

```diff
@@ -26,36 +26,52 @@ let haveWarnedDeprecations = false;
  * default config from `../../constants/default-config`.
  */
 const getConfig = memoizeOne(function getConfig() {
-  const getJaegerUiConfig = window.getJaegerUiConfig;
-  if (typeof getJaegerUiConfig !== 'function') {
-    if (!haveWarnedFactoryFn) {
-      // eslint-disable-next-line no-console
-      console.warn('Embedded config not available');
-      haveWarnedFactoryFn = true;
-    }
-    return { ...defaultConfig };
-  }
-  const embedded = getJaegerUiConfig();
+  const capabilities = getCapabilities();
+  const config = {...defaultConfig, storageCapabilities: capabilities};
+
+  const embedded = getUiConfig();
   if (!embedded) {
-    return { ...defaultConfig };
+    return config;
   }
   // check for deprecated config values
   if (Array.isArray(deprecations)) {
     deprecations.forEach(deprecation => processDeprecation(embedded, deprecation, !haveWarnedDeprecations));
     haveWarnedDeprecations = true;
   }
-  const rv = { ...defaultConfig, ...embedded };
+  const rv = { ...config, ...embedded};
```

Comment:
This can potentially override storage capabilities - probably ok, although might be unexpected if done by accident.